R/jointScoreTest.R

```diff
@@ -51,20 +51,20 @@ jointScoreTest <- function(nullmod, G) {  # # Check rownames/colnames match.
   # Create fixed effects data frame.
   fixef <- data.frame(
     Est = as.vector(beta),
-    SE = se,
+    Est.SE = se,
```

Comment:
@smgogarten in the null model "fixef" output, we just call this SE. I'm guessing that's where @amstilp took the naming from. See the file prepareOutputForNullModel.R

Ultimately, it would be better if these were _all_ consistent, but I'm not sure which way we should go with this one if we don't want to change one of the older ones.

Reply:
Good point, this should match the null model. I changed it back.

Reply:
To Adrienne's point, we could make them all be Est.SE. We do plan to refactor the null model output format in the near future (might make that a Q4 2020 SBG update). If we think it's best to make them all be Est.SE, then we should make the jointScoreTest one be that now. I have mixed feelings if that is the right way to go. I think the null model output was named SE to be like lm. In the assoc output, we have both a Score.SE and Est.SE, so we needed to specify which was which.